### ISSUE TITLE
Fill in missing public liaison numbers and name

### ISSUE THREAD

**Issue by ramirezg** [18F/foia-hub#421]
Make sure we have public liaison numbers and names for all departments. 

***Missing numbers***
```
Missing public liaison number: Patricia Blumenthal National Finance Center data/USDA.yaml
Missing public liaison number: Veronica Herkshan Office of the Special Trustee data/DOI.yaml
```

***Missing names***
```
Missing public liaison name: 202-208-0954 data/DOI.yaml
Missing public liaison name: 816-268-8000 data/NARA.yaml
Missing public liaison name: 202-358-0718 data/NASA.yaml
Missing public liaison name: 703-518-6540 data/NCUA.yaml
Missing public liaison name: 202-682-5418 data/NEA.yaml
Missing public liaison name: 202-482-9203 data/OGE.yaml
Missing public liaison name: 703-545-5993 data/SIGAR.yaml
Missing public liaison name: 202-317-3475 data/Treasury.yaml
Missing public liaison name: 202-673-3916 data/US ADF.yaml
Missing public liaison name: 312-751-4350 data/US RRB.yaml
Missing public liaison name: 202-632-7465 data/VA.yaml

```

**Comment by ramirezg** [comment]
Looks like offices with no names might have generic number that don't actually reach FOIA ei [Archives](http://www.archives.gov/kansas-city/contacts.html)

**Comment by ramirezg** [comment]
The offices in [this](https://gist.github.com/ramirezg/8ff1410d2fc701f78986) list are missing everything all public liaisons.

**Comment by ramirezg** [comment]
Fixed with: 18F/foia/pull/102 
I only corrected the Agencies with incorrect numbers for public liaisons. If the name wasn't there originally I didn't add it, b/c it's not critical.
